package/MDAnalysis/coordinates/memory.py

```diff
@@ -287,9 +287,11 @@ def __init__(self, coordinate_array, order='fac',
         self._read_next_timestep()
 
     @staticmethod
-    def parse_n_atoms(filename):
+    def parse_n_atoms(filename, **kwargs):
         # assume filename is a numpy array
-        return filename.shape[0]
+        # default order 'fac'
+        order = kwargs.get('order', 'fac')
+        return filename.shape[order.find('a')]
```

Comment:
This should be explicit about the `order` keyword argument in the function definition.  `parse_n_atoms(filename, order='fac', **kwargs)` This doesn't affect the API of the others but we are explicit in the function definition.

Reply:
ok fixed

Reply:
Thanks